spec/helper.rb

```diff
@@ -45,6 +45,12 @@
 ActiveJob::Base.logger = Delayed.logger
 ActiveRecord::Migration.verbose = false
 
+if ActiveRecord.respond_to?(:default_timezone=)
+  ActiveRecord.default_timezone = :utc
+else
+  ActiveRecord::Base.default_timezone = :utc
+end
```

Comment:
Just noting that I didn't touch this config outside of `spec/` -- it resolves an issue where CI's`:local` is different from my laptop's, and we want to test the generated SQL, which includes specific timestamps.